plugins/com.contrastsecurity.ide.eclipse.ui/src/com/contrastsecurity/ide/eclipse/ui/internal/views/FilterDialog.java

```diff
@@ -0,0 +1,714 @@
+		severityCompositeContainer.setLayout(new GridLayout(2, false));
+
+		label = new Label(severityCompositeContainer, SWT.NONE);
+		gd = new GridData(SWT.FILL, SWT.CENTER, false, false);
+		label.setLayoutData(gd);
+		label.setText("Severity");
+
+		Composite severityComposite = new Composite(severityCompositeContainer, SWT.NONE);
+		severityComposite.setLayout(new GridLayout(3, false));
+
+		severityLevelNoteButton = new Button(severityComposite, SWT.CHECK);
+		severityLevelNoteButton.setText("Note");
+
+		severityLevelMediumButton = new Button(severityComposite, SWT.CHECK);
+		severityLevelMediumButton.setText("Medium");
+
+		severityLevelCriticalButton = new Button(severityComposite, SWT.CHECK);
+		severityLevelCriticalButton.setText("Critical");
+
+		severityLevelLowButton = new Button(severityComposite, SWT.CHECK);
+		severityLevelLowButton.setText("Low");
+
+		severityLevelHighButton = new Button(severityComposite, SWT.CHECK);
+		severityLevelHighButton.setText("High");
+	}
+
+	private void createStatusSection(Composite container) {
+
+		GridData gd = new GridData(SWT.FILL, SWT.FILL, true, true);
+
+		Composite statusCompositeContainer = new Composite(container, SWT.NONE);
+		statusCompositeContainer.setLayout(new GridLayout(2, false));
+
+		label = new Label(statusCompositeContainer, SWT.NONE);
+		gd = new GridData(SWT.FILL, SWT.CENTER, false, false);
+		label.setLayoutData(gd);
+		label.setText("Status");
+
+		Composite statusComposite = new Composite(statusCompositeContainer, SWT.NONE);
+		statusComposite.setLayout(new GridLayout(3, false));
+
+		statusAutoRemediatedButton = new Button(statusComposite, SWT.CHECK);
+		statusAutoRemediatedButton.setText("Auto-Remediated");
+
+		statusNotAProblemButton = new Button(statusComposite, SWT.CHECK);
+		statusNotAProblemButton.setText("Not a Problem");
+
+		statusFixedButton = new Button(statusComposite, SWT.CHECK);
+		statusFixedButton.setText("Fixed");
+
+		statusConfirmedButton = new Button(statusComposite, SWT.CHECK);
+		statusConfirmedButton.setText("Confirmed");
+
+		statusRemediatedButton = new Button(statusComposite, SWT.CHECK);
+		statusRemediatedButton.setText("Remediated");
+
+		statusBeingTrackedButton = new Button(statusComposite, SWT.CHECK);
+		statusBeingTrackedButton.setText("Being Tracked");
+
+		statusSuspiciousButton = new Button(statusComposite, SWT.CHECK);
+		statusSuspiciousButton.setText("Suspicious");
+
+		statusReportedButton = new Button(statusComposite, SWT.CHECK);
+		statusReportedButton.setText("Reported");
+
+		statusUntrackedButton = new Button(statusComposite, SWT.CHECK);
+		statusUntrackedButton.setText("Untracked");
+	}
+
+	private void createLastDetectedSection(Composite container) {
+		GridData gd = new GridData(SWT.FILL, SWT.FILL, true, true);
+		Composite lastDetectedCompositeContainer = new Composite(container, SWT.NONE);
+		lastDetectedCompositeContainer.setLayout(new GridLayout(3, false));
+
+		label = new Label(lastDetectedCompositeContainer, SWT.NONE);
```

Comment:
Create a function where you can init the label objects, this way you will recycle code.

Reply:
Done.